src/app/(proper_react)/(redesign)/(public)/LandingView.test.tsx

```diff
@@ -812,6 +813,36 @@ describe("When Premium is available", () => {
     );
   });
 
+  it("shows the scan limit and waitlist cta when the DisableOneRepScans flag is on", () => {
```

Comment:
For extra confidence it would be neat to have this test, but for a non-US user (with the flag enabled), that checks that the description is _not_ in the doc, just to be sure.